spec/unit/locale_spec.rb

```diff
@@ -31,6 +36,19 @@
     it "process_locale_nil" do
       expect(SgtnClient::LocaleUtil.process_locale(nil)).to eq SgtnClient::Config.configurations.default
     end
+
+
+    it "default" do
+      env = SgtnClient::Config.default_environment
+      config_lang = SgtnClient::Config.configurations[env]["default_language"]
+      if config_lang.nil? || config_lang == 'en'
```

Comment:
Never use "if" in test cases. You should know the actual value in tests. Set the values per case and create 1 test per test case. See "get_source_locale" and "get_source_locale_sourceIsNil" in [spec/unit/locale_spec.rb](https://github.com/vmware/singleton/pull/1551/files#diff-36e605de231d4c6177003c6936c7e380df52bb2d68591fdf04dc2381bd023baf)

Reply:
done